src/commands/map.ts

```diff
@@ -70,6 +72,12 @@ export default class Map extends Command {
 
   public static flags = {
     ...Command.flags,
+    timeout: oclifFlags.integer({
+      char: 't',
+      required: false,
+      description: `Opration timeout in seconds. If not provided, it will be set to ${DEFAULT_POLLING_TIMEOUT_SECONDS} seconds. Usefull for large APIs.`,
```

Comment:
typo

```suggestion
      description: `Operation timeout in seconds. If not provided, it will be set to ${DEFAULT_POLLING_TIMEOUT_SECONDS} seconds. Useful for large API documentations.`,
```